app/views/notes/show.html.erb

```diff
@@ -78,6 +78,17 @@
   <div style="overflow:hidden;" id="content">
   <%= raw auto_link(@node.latest.render_body, :sanitize => false) %>
   </div>
+  
```

Comment:
This and associated javascript includes should be moved into a partial, which can be called from note, wiki, or question.